conda_build/post.py

```diff
@@ -1201,25 +1199,20 @@ def _lookup_in_prefix_packages(
     elif len(precs_in_reqs) == 0 and len(precs) > 0:
         _print_msg(
             errors,
-            "{}: {} found in {}{}".format(
-                msg_prelude, n_dso_p, [prec.name for prec in precs], and_also
-            ),
+            f"{msg_prelude}: {n_dso_p} found in {[str(prec) for prec in precs]}{and_also}",
```

Comment:
This upgrades from name to the full str representation. i.e. from `python` to `defaults::python-3.12.0-h123abc_0`, I think? Is that intentional?

Reply:
yes, this is now more detailed (but still less verbose than `str(precs)`) than before

the change was inspired by the fix below on line 1214-1215, where `precs_in_reqs: List[PackageRecord]` was cast to string resulting in a very verbose output, see https://github.com/conda/conda-build/pull/5108#pullrequestreview-1782145301

Reply:
I think it's good that this is now more consistent with the full spec which was already output at https://github.com/conda/conda-build/pull/5108/files#diff-5073339a2e88f80714f82a001af334869aa5df9627b2f00ffd18d7a2035f537eR1190 .